server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/GetMessagesMethodStepdefs.java

```diff
@@ -235,6 +235,11 @@ public void appendMessageWithSpecialCase(String messageName, String mailbox) thr
         appendMessage(messageName, "eml/htmlWithLongAndComplicatedContent.eml");
     }
 
+    @Given("^the user has a message \"([^\"]*)\" in \"([^\"]*)\" mailbox with iso charset")
+    public void appendMessageWithEncoding(String messageName, String mailbox) throws Throwable {
```

Comment:
Yes, why it cannot? It looks like the other test methods. Do I need to change them all to "Exception" instead of?